### ISSUE TITLE
Prefect Graph UI not updating flow run times accurately

### ISSUE THREAD

**Issue by japerry911** [PrefectHQ/prefect-ui-library#2260]
In the below example, my flow says it's been running for 1h 15 minutes, but in reality it has been running for about 1h 50m. I have been witnessing this error for a few days, so it does not appear one-off.

I tried to refresh the page, but the name/time do not change in the main dashboard graph. It does appear to change in the flow runs item list, and displays accurate time there, just not in the flow runs dashboard graph (image 1) or the specific flow run page (image 2). Image 3 shows correct time duration in flow runs items list.

Browser: Chrome
Environment Prefect Cloud | app.prefect.cloud

Let me know if this doesn't make sense, or if you would like more details. 

Thank you!

![Screenshot 2024-03-26 at 2 22 05 PM](https://github.com/PrefectHQ/prefect-ui-library/assets/21957238/b93370b9-1b6d-4653-8617-4daed6c6d623)

![Screenshot 2024-03-26 at 2 24 00 PM](https://github.com/PrefectHQ/prefect-ui-library/assets/21957238/3fe67fe3-9330-44f7-8a16-99ca9214f6bf)

![Screenshot 2024-03-26 at 2 24 47 PM](https://github.com/PrefectHQ/prefect-ui-library/assets/21957238/48c18abe-42e7-4f4e-9451-08c4938d294a)

**Comment by zhen0** [comment]
Thanks for the issue @japerry911 - we'll look into this further.

**Comment by pleek91** [comment]
@japerry911 The only thing I was able to reproduce was the popover from the run graph wouldn't have the latest duration if I closed it and then opened it again later (without refreshing the page). I'll work on a fix for that piece. 

The other issues I wasn't able to reproduce. The flow run page seemed to update properly to reflect the duration. And the run graph would move the blue dot up as the duration got larger. 

I tried this with several flow runs. One which ran for an hour before completing.
